packages/create-bison-app/template/components/Link.tsx

```diff
@@ -0,0 +1,102 @@
+import type { LinkProps, NextLinkAs } from 'types';
```

Comment:
Is the module for this import right? I'm getting errors when running `yarn dev:typecheck`.

Reply:
@kgajera It turned out to be an issue of the types I defined in `types.ts` being in the gitignore file. And even then, the file would have ended up overwritten by GraphQL Codegen generated types (which is why the file is ignored to begin with 😄 ).

The fix was just to move the types I defined for the component in a different file. I ended up declaring them inside the `Link.tsx` component file itself since they weren't being used in any other modules.